_integration_tests/src/fixture_providers/ioc_module.ts

```diff
@@ -21,6 +21,7 @@ export function registerInContainer(container: IContainer): void {
         'ConsumerApiNotificationService',
         'ConsumerApiProcessModelService',
         'ConsumerApiUserTaskService',
+        'ConsumerApiTaskService',
```

Comment:
Das hier würde dann, wenn der Rest implementiert ist wie folgt heißen:

```suggestion
        'ConsumerApiFlowNodeInstanceService',
```